python/dgl/data/minigc.py

```diff
@@ -1,13 +1,17 @@
 """A mini synthetic dataset for graph classification benchmark."""
-import math
+import math, os
 import networkx as nx
 import numpy as np
 
+from .dgl_dataset import DGLDataset
+from .. import backend as F
+from .utils import save_graphs, load_graphs, makedirs
 from ..graph import DGLGraph
+from ..graph import batch as graph_batch
 
 __all__ = ['MiniGCDataset']
 
-class MiniGCDataset(object):
+class MiniGCDataset(DGLDataset):
```

Comment:
Do we need examples in docstring?

Reply:
Eventually we do. Maybe we copy the docstring in hackmd here later, or reversely.